codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -527,10 +527,19 @@ public void onUnknownFrame(ChannelHandlerContext ctx, byte frameType, int stream
             onUnknownFrame0(ctx, frameType, streamId, flags, payload);
         }
 
+        /**
+         * Helper method for determining whether or not to ignore inbound frames. This is done after a {@code GOAWAY}
+         * frame was sent by this endpoint and the stream: <br/> <ol> <li>was created remotely and </li> <li>is known by
```

Comment:
`is known by this endpoint`.  Can we clarify this and be more explicit with the `known` terminology?  Something like the following?

```
A stream is considered to be created after a go away is sent if the following conditions hold:
<ul>
<li>A {@code GOAWAY} must have been sent by this endpoint</li>
<li>The {@code streamId} must identify a legitimate stream id for the remote peer to be creating</li>
<li>{@code streamId} is greater than the Last Known Stream ID which was sent by this endpoint in the last {@code GOAWAY} frame</li>
</ul>
```